sep/2-op-extended-pause/.env

```diff
@@ -1,4 +1,2 @@
 ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
-FOUNDATION_SAFE="TBD"
-SAFE_NONCE="0"
```

Comment:
OK, replaced it with the current nonce.

```
cast call 0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B 'nonce()'
0x0000000000000000000000000000000000000000000000000000000000000003
```

Reply:
This error is because they require that env var and don't use `envOr`, but there's a PR that fixed this here: https://github.com/base-org/contracts/pull/55/files

So we should probably update the base contracts dep to a newer version

Reply:
I will try that, but if anything else breaks will leave it for future work.